docs/rules/sort-imports.md

```diff
@@ -136,6 +138,34 @@ import c from 'baz.js';
 
 Default is `false`.
 
+### `ignoreDeclarationSort`
+
+Ignores the sorting of import declaration statements.
+
+Examples of **incorrect** code for this rule with the default `{ "ignoreMemberSort": false }` option:
```

Comment:
I think this should say `"ignoreDeclarationSort"` based on the following examples.